analysis/neurips_bt_magical_full.py

```diff
@@ -0,0 +1,230 @@
+"""Script MAGICAL results as LaTeX for the NeurIPS benchmark track.
+
+Like `neurips_bt_magical.py`, but prints out full results for all variants.
```

Comment:
It seems `neurips_bt_magical.py` and `neurips_bt_magical_full.py` have some common functions. Can some be imported to simplify the code?